src/pages/Scholarships.jsx

```diff
@@ -33,6 +35,10 @@ function ScholarshipsPage() {
 
   const classes = useStyles();
 
+  const theme = createMuiTheme({
+    spacing: 8,
+  });
```

Comment:
This is not how themes work. This creates a new theme instead of using the existing theme. You can modify line 14 to take in the theme and use the default theme instead:
```js
const useStyles = makeStyles((theme) => ({
```